client/components/purchase-detail/index.jsx

```diff
@@ -99,7 +98,7 @@ export default class PurchaseDetail extends PureComponent {
 					<div className="purchase-detail__image">{ this.renderIcon() }</div>
 					<div className="purchase-detail__text">
 						<h3 className="purchase-detail__title">{ title }</h3>
-						<div className="purchase-detail__description">{ preventWidows( description ) }</div>
```

Comment:
On a site with a Business plan, go to Upgrades > Plans:

<img width="485" alt="Screenshot 2025-05-12 at 1 49 15 pm" src="https://github.com/user-attachments/assets/7065d22f-c1bd-4622-aed3-3e27dbe47193" />

The element is a `div` but its parent `purchase-detail__text` has custom styles that cascade down.